File: backend/app/api/v2/schemas/billing.py
from pydantic import BaseModel


class BillingOverview(BaseModel):
    """The org's credit budget. There are no plans or subscriptions — the
    organization is the budget holder and every member draws from this one
    pool. `total = -1` means unlimited.
    """
    intake_total: int = 0
    intake_used: int = 0
    intake_topup: int = 0
    interview_total: int = 0
    interview_used: int = 0
    interview_topup: int = 0

File: recruiter-app/src/domain/billing.ts
// The organization holds one credit budget that every member draws from.
// There are no plans or subscriptions — an admin sets the cap from the admin
// portal. A total of -1 means unlimited.
export interface BillingOverview {
  intake_total: number;
  intake_used: number;
  interview_total: number;
  interview_used: number;
}
